node/module_test.ts

```diff
@@ -131,7 +131,6 @@ Deno.test("EventEmitter is exported correctly", () => {
 Deno.test("Require .mjs", () => {
   assertThrows(
     () => require("./testdata/inspect.mjs"),
-    undefined,
```

Comment:
This has the same issue as the above one (`crypto_browserify/evp_bytes_to_key_test.ts`)